app/views/users/index.html.erb

```diff
@@ -1,8 +1,12 @@
-<h1>List of all static users:</h1>
-<ul>
-  <li>User 1</li>
-  <li>User 2</li>
-  <li>User 3</li>
-  <li>User 4</li>
-  <li>User 5</li>
-</ul>
+<div>
+  <ul>
+  <% @users.each do |user| %>
+  <li>
+  <div>
+    <h1><%= user.name %></h1>
+    <h4>Number of posts: <%= user.posts.count %></h4>
+  </div>
+  </li>
+  <% end %>
+  </ul>
+</div>
```

Comment:
| @Alibaba2023 User Index | Microverse User Index |
|-|-|
| ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/2f312413-6d4b-43c6-8d6f-71c2688ddd0f)) | ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/f7c18d76-78e3-4c0d-af36-a9392feb802a) |

- [ ] It would be nice if you follow the given wireframe because here should be the image of the user which we insert into the database 🤔 to follow the project requirements 👍
- [ ] Kindly put the Name and posts counter at the same place as you can see in the given wireframe 👀 to follow the requirements to make your design look professional 😊